cdap-ui/app/main.js

```diff
@@ -310,6 +310,8 @@ angular
       getVersion();
     }
 
+    $scope.copyrightYear = new Date().getFullYear();
```

Comment:
that's probably because 2015 was the first year of the new UI? @awholegunch can you confirm?